README.md

```diff
@@ -33,11 +33,15 @@ directories will exist:
 Auto-generated developer documentation can be found at our corresponding
 [doclets site](https://doclets.io/bookbrainz/bookbrainz-site/master). Our
 contributing guide can be found [here](CONTRIBUTING.md).
-
-
-<br/>
 <br/>
+## Testing subdomain
 
+A separate subdomain for the whole purpose of testing can be found [here](https://test.bookbrainz.org). 
```

Comment:
I'd suggest "[...] for the purpose of testing and rolling out beta features [...]"